docs/user/debugging_tips.md

```diff
@@ -65,11 +65,14 @@ The following profiles are available:
  24 profiles, 91 plugins
 ```
 
-Log into the host running MicroShift and execute the following command to generate an obfuscated
-report that should not contain sensitive information (in this example only microshift plugins are
-enabled. Your system's output may vary):
+Log into the host running MicroShift and execute the following command to generate a
+report (in this example only microshift plugins are enabled. Your system's output may vary):
+
+> Note: If necessary, sensitive data in the sos report may be obfuscated with the `--clean` flag.  `sosreport` includes
+> kubernetes logs, which may be large.  Obfuscation processes on large files can take as long as several hours.
+
```

Comment:
Can we add a mentioned about the new `microshift-sos-report` script here?

Reply:
WDYT about adding `--clean` is an optional flag to the script?

Reply:
We didnt add flags to the script because is like an alias for the sos. The intention is easing the use for people unfamiliar with sos, so that we get at least the bare minimum.